lib/oli_web/controllers/page_delivery_controller.ex

```diff
@@ -107,6 +107,7 @@ defmodule OliWeb.PageDeliveryController do
       next_page: context.next_page,
       title: context.page.title,
       graded: context.page.graded,
+      activity_count: context.activity_count,
```

Comment:
Replace this with `activity_count: map_size(context.activities),`

Reply:
Replacement done